src/modules/vulnerability/components/card/ReportServiceCard.vue

```diff
@@ -1,124 +1,150 @@
               <div class="col-3 text-weight-medium">
                 {{ $t('report.vulnerabilityCategory.version') }}
               </div>
-              <div class="col">{{ vulnerabilityGroup.version }}</div>
+              <div class="col">{{ assets.operating_system?.version }}</div>
             </div>
             <div class="row q-pl-lg flex-1 items-center">
               <div class="col-3 text-weight-medium">
                 {{ $t('report.vulnerabilityCategory.precision') }}
               </div>
-              <div class="col">{{ vulnerabilityGroup.precision }}</div>
+              <div class="col">{{ assets.operating_system?.accuracy }}</div>
             </div>
-          </template>
-          <template v-else-if="category === ReportVulnerabilityGroupCategory.SERVICE">
-            <div class="row">
-              <div class="col-6">
-                <div class="row q-pl-lg flex-1 items-center">
-                  <div class="col-3 text-weight-medium">
-                    {{ $t('report.vulnerabilityCategory.portNumber') }}
+          </div>
+        </q-card-section>
+
+        <q-separator vertical />
+        <q-card-section class="col-6 position-relative">
+          <q-btn square size="md" flat dense icon="exit_to_app" class="exit-button"></q-btn>
+          <div class="title">{{ $t('report.vulnerabilityCategory.details') }}</div>
+          <div class="row text-center full-width">
+            <div class="col-12 text-weight-bold text-body1">
+              {{ assets.operating_system?.total_vulnerabilities_count }}
+            </div>
+            <div class="col-12 text-body2">
+              {{ $t('report.vulnerabilityCategory.vulnerabilities') }}
+            </div>
+          </div>
+          <div class="row q-pa-none q-pt-md">
+            <template v-for="level in vulnerabilityLevels" :key="level">
+              <div class="vulnerability-count col" :class="level.toLocaleLowerCase()">
+                {{ assets.operating_system?.[level] }}
+              </div>
+            </template>
+          </div>
+        </q-card-section>
+      </q-card-section>
+    </q-card>
+    <h5 class="q-ma-none q-my-sm text-weight-medium">Services</h5>
+    <template v-if="assets.services?.length">
+      <template v-for="service in assets.services" :key="service">
+        <q-card flat class="bg-grey-2 q-pa-sm q-my-md" style="border-radius: 1em">
+          <q-card-section horizontal class="justify-between">
+            <q-card-section class="col-6 flex column">
+              <div class="title">{{ service.name }}</div>
+              <div class="row">
+                <div class="col-6">
+                  <div class="row q-pl-lg flex-1 items-center">
+                    <div class="col-3 text-weight-medium">
+                      {{ $t('report.vulnerabilityCategory.portNumber') }}
+                    </div>
+                    <div class="col">{{ service.port }}</div>
                   </div>
```

Comment:
This HTML block and class properties are leading to the "Port number" alias and value being rendered way too close to each other

<img width="870" height="477" alt="Image" src="https://github.com/user-attachments/assets/12dfd832-0ecc-4d2d-ac31-f03c2654601b" />


```suggestion
                  <div class="row q-pl-lg flex-1 items-center q-col-gutter-x-md">
                    <div class="col-3 text-weight-medium">
                      {{ $t('report.vulnerabilityCategory.portNumber') }}
                    </div>
                    <div class="col">{{ service.port }}</div>
</div>
```